Protacon.RxMq.AzureServiceBusLegacy/packagerefs.net48.targets

```diff
@@ -0,0 +1,6 @@
+﻿<Project>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Reactive" Version="5.0.0" />
```

Comment:
Mihin tätä tarvitaan?

Reply:
Targeting net48, we want to have latest Newtonsoft and it depends on System.Reactive, hence the updates.